content/client-lib-development-guide/features.textile

```diff
@@ -350,14 +347,14 @@ h3(#realtime-channel). Channel
 *** @(RTL6c2)@ If the connection is @CONNECTING@ or @DISCONNECTED@, and @ClientOptions#queueMessages@ has not been explicitly set to false, then the message will be queued and delivered as soon as the connection state returns to @CONNECTED@
 *** @(RTL6c3)@ Else an exception is raised preventing the message publishing
 ** @(RTL6d)@ Messages are delivered using a single @ProtocolMessage@ where possible by bundling in all messages for that channel into the @ProtocolMessage#messages@ array. However, a yet to be implemented feature should limit the total number of messages bundled per @ProtocolMessage@ based on the default max message size, and would raise an exception if any single message exceeds that limit
-** @(RTL6e)@ Unauthenticated clients without a @clientId@ (any @clientId@ is permitted):
+** @(RTL6e)@ Unauthenticated clients using basic auth without a @clientId@ (i.e. any @clientId@ is permitted):
```

Comment:
As above, not sure token auth without a `clientId` allows any `clientId` to be used, at least as far as I was concerned